Ical.Net/DataTypes/PeriodList.cs

```diff
@@ -69,7 +69,10 @@ public override void CopyFrom(ICopyable obj)
 
         foreach (var p in list)
         {
-            Add(p.Copy<Period>());
+            if (p != null)
```

Comment:
By not copying null values the copy will not be equal to the original. Maybe to do something like `Add(p?.Copy<Period>())`?